### PR TITLE
Add a better design to the plugins command

### PR DESCRIPTION
**Pull request by Kaooot**

### CODE REVIEW

File: gomint-server/src/main/java/io/gomint/server/command/gomint/PluginsCommand.java [hunk 1]
```diff
@@ -26,6 +26,11 @@
     @Override
     public CommandOutput execute(CommandSender commandSender, String alias, Map<String, Object> arguments) {
         Map<String, Plugin> plugins = GoMint.instance().getPluginManager().getPlugins();
+
+        if (plugins.size() == 0) {
+            return CommandOutput.failure("There aren't any plugins on this server.");
```
Comment (Soupertonic) [inline gomint-server/src/main/java/io/gomint/server/command/gomint/PluginsCommand.java]: Please change the message to `No plugins were loaded.`

**Review by Soupertonic — CHANGES_REQUESTED** [review]

File: gomint-server/src/main/java/io/gomint/server/command/gomint/PluginsCommand.java [hunk 2]
```diff
@@ -26,6 +26,19 @@
...
-        return CommandOutput.successful(String.join(", ", plugins.keySet()));
+
+        if (plugins.size() == 0) {
+            return CommandOutput.failure("No plugins were loaded.");
+        }
+
+        StringBuilder stringBuilder = new StringBuilder("Plugins (" + "§a" + plugins.size() + "§r): ");
+
+        for (Plugin plugin : plugins.values()) {
+            stringBuilder.append("§a").append(plugin.getName()).append(" v").append(plugin.getVersion().toString()).append("§r, ");
```
Comment (geNAZt) [inline gomint-server/src/main/java/io/gomint/server/command/gomint/PluginsCommand.java]: Plugin version should not be exposed for security reasons

File: gomint-server/src/main/java/io/gomint/server/command/gomint/PluginsCommand.java [hunk 3]
```diff
@@ -26,6 +26,19 @@
     @Override
     public CommandOutput execute(CommandSender commandSender, String alias, Map<String, Object> arguments) {
         Map<String, Plugin> plugins = GoMint.instance().getPluginManager().getPlugins();
-        return CommandOutput.successful(String.join(", ", plugins.keySet()));
+
+        if (plugins.size() == 0) {
+            return CommandOutput.failure("No plugins were loaded.");
+        }
+
+        StringBuilder stringBuilder = new StringBuilder("Plugins (" + "§a" + plugins.size() + "§r): ");
```
Comment (geNAZt) [inline gomint-server/src/main/java/io/gomint/server/command/gomint/PluginsCommand.java]: Remove concat between ( and §a

File: gomint-server/src/main/java/io/gomint/server/command/gomint/PluginsCommand.java [hunk 4]
```diff
@@ -26,6 +26,19 @@
     @Override
     public CommandOutput execute(CommandSender commandSender, String alias, Map<String, Object> arguments) {
         Map<String, Plugin> plugins = GoMint.instance().getPluginManager().getPlugins();
-        return CommandOutput.successful(String.join(", ", plugins.keySet()));
+
+        if (plugins.size() == 0) {
```
Comment (geNAZt) [inline gomint-server/src/main/java/io/gomint/server/command/gomint/PluginsCommand.java]: Use isEmpty()

**Review by geNAZt — CHANGES_REQUESTED** [review]

File: gomint-server/src/main/java/io/gomint/server/command/gomint/PluginsCommand.java [hunk 5]
```diff
@@ -26,6 +26,19 @@
...
+            return CommandOutput.failure("No plugins were loaded.");
+        }
+
+        StringBuilder stringBuilder = new StringBuilder("Plugins (§a" + plugins.size() + "§r): §a");
+
+        for (Plugin plugin : plugins.values()) {
+            stringBuilder.append(plugin.getName()).append("§r, ");
+        }
+
+        stringBuilder.setLength(stringBuilder.length() - 2);
```
Comment (Janmm14) [inline gomint-server/src/main/java/io/gomint/server/command/gomint/PluginsCommand.java]:
```suggestion
        stringBuilder.setLength(stringBuilder.length() - 4);
```

Comment (geNAZt) [reply gomint-server/src/main/java/io/gomint/server/command/gomint/PluginsCommand.java]: To get rid of the last §r?

Comment (Janmm14) [reply gomint-server/src/main/java/io/gomint/server/command/gomint/PluginsCommand.java]: Yes, its not making a visual difference, but we don't need it.

**Review by geNAZt — APPROVED** [review]

**Review by Soupertonic — APPROVED** [review]

**Pull request merged**